backend/test/tutti-pizza.parser.spec.js

```diff
@@ -0,0 +1,21 @@
+let should = require('should');
+require('mocha');
+const { TuttiPizza } = require('../pizzas-providers/parse-tutti-pizza.js');
+
+// Not a real test but more an executor.
+// TODO to be improved
```

Comment:
What do you mean by

> Not a real test but more an executor

?

Reply:
At the beginning, no assertion was made. As far as I have a bit more tested the functionnality I've removed this no more applyable comment.